CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1210
+
+* Staging release date: TBD
+* Production APM-injected release date: TBD
+* Production Standalone release date: TBD
+
+
+### PageHide events are now accounted for during page unload events
+`PageHide` events are used to query CLS values, which led to inconsistencies if the `pageHide` value was never set.  Now when `pageUnload` fires, if `pageHide` has not already been set, it will set at to the time of unload. 
+
+### Perfect Cumulative Layout Scores (CLS) are now recorded as 0
+Perfect CLS scores were being ignored, because a score was only recorded when content shifted.  This change reports perfect scores as 0, fixing inconsistent CLS queries.
+
+### Record fetch calls as metrics
+Fetch calls are currently only recorded as AjaxRequest events with SPA browser interactions. This change records fetch calls as AJAX metrics, which will make them visible in the AJAX UI charts.
+
+### Updated configuration test for checking default URL values
```

Comment:
In the past we have only included changes to the agent code that is released to customers. I would say let's leave this change to tests out, but I am open to other opinions. :)

Reply:
is the `changelog` ever referenced by anyone except customers?